scripts/runner.py

```diff
@@ -0,0 +1,95 @@
+"""
+Example script for running a set of metrics asynchronously
+"""
+
+import pathlib
+from pathlib import Path
+
+from celery.exceptions import NotRegistered
+from loguru import logger
+from ref_celery.app import create_celery_app
+from ref_core.env import env
+from ref_core.metrics import Configuration, TriggerInfo
+
+config = {
```

Comment:
The parent process will need to track the jobs that have been declared as worker join and leave the cluster.  The workers may declare the list of jobs that they provide and their dependencies at run-time

Perhaps the parent process needs to know which providers to listen to some way of whitelisting/blacklisting metrics.